src/reference/informative/informC08_interpretation_of_mathematics2.rst

```diff
@@ -0,0 +1,216 @@
+            <variable name="runs" units="dimensionless" />
+            <math>
+              <!-- This statement sets Dick's score to 0 for all time.  
+                  He's a dog.  He can't use a cricket bat. -->
+              <apply><eq/>
+                <ci>runs</ci>
+                <cn cellml:units="dimensionless">0</cn>
+              </apply>
+            </math>
+          </component>
+
+          <component name="Harry">
+            <variable name="time" units="minute" />
+            <variable name="runs" units="dimensionless" />
+            <math>
+              <!-- This statement represents DickTheDog running away with the ball, 
+                  enabling Harry to score an ever-increasing number of runs. -->
+              <apply><eq/>
+                <ci>runs</ci>
+                <apply><times/>
+                  <cn cellml:units="per_minute">10</cn>
+                  <ci>time</ci>
+              </apply>
+            </math>
+          </component>
+
+          <!-- A connection is established between Harry and Dick to enable sharing of their run tally. -->
+          <connection component_1="Harry" component_2="DickTheDog" >
+            <map_variables variable_1="dicks_runs" variable_2="runs" />
+          </connection>
+          ...
+        </model>
+
+
+    .. container:: heading3
+
+      Understanding how and when the mathematics *doesn't* work
+
+    It's possible to write valid CellML that does not represent valid mathematics.
+    You can think of this like correctly spelling a set of words which together do not form a meaningful sentence.
+    Some examples of valid versus valid-but-nonsense :code:`math` elements' contents are shown below.
+
+    Simple over-definition is valid, but will result in behaviour that is hard to interpret meaningfully:
+
+    .. math::
+
+      x = 0
+
+      x = 1
+
+    Complicated over-definition is likewise valid:
+
+    .. math::
+
+      x + y = 1
+
+      x - y = 3
+
+      x * y = 12
+
+    Redundant information is valid, but (well) redundant:
+
+    .. math::
+
+      x = 1
+
+      x = 1
+
+      x = 1
+
+    Under-definition at a localised component level is both valid and useful, as you may need to connect to other components in order to know the value of the variables the maths statements are using.
+    Models which *overall* have insufficient definition are also valid, but clearly won't be useful or solvable.
+
+    .. math::
+
+      x = y + z
+
+    Unsolvable models and "bad" maths is valid CellML:
+
+    .. math::
+
+      x = 1 / 0
+
+      x = \sqrt{-1}
+
+    Conflicting information arising from initialising variables which are not state variables will have an outcome which depends on how the implementation software interprets the condition.  
+    It is not invalid CellML, but - as with other forms of over-definition - may not result in the same interpretation between software implementations. 
+
+    .. math::
+
+      x = 1
+      x(0) = 2
```

Comment:
Put an empty line between the two mathematical statements as they are currently rendered  as "𝑥=1𝑥(0)=2"

Reply:
Done (somewhere ... )